README.md

```diff
@@ -465,6 +468,54 @@ extensions such as `import com.klaviyo.pushFcm.KlaviyoRemoteMessage.body` to acc
 We also provide an `Intent.appendKlaviyoExtras(RemoteMessage)` extension method, which attaches the data to your
 notification intent that the Klaviyo SDK requires in order to track opens when you call `Klaviyo.handlePush(intent)`.
 
+## In App Forms
+
+In-app forms are messages displayed to mobile app users while they are actively using your app. 
+You can create new in-app forms in a drag-and-drop editor in the Sign-Up Forms tab in Klaviyo.
+
+[//]: # (TODO -- add help doc link)
+
+### Prerequisites
+- Version 3.2.0 or higher
+- Klaviyo `analytics` and `forms` packages
+
+### Setup
+To display in-app forms add the following code to your application. 
+
+```kotlin
+import com.klaviyo.analytics.Klaviyo
+import com.klaviyo.forms.registerForInAppForms
+
+// You can register as soon as you've initialized
+Klaviyo
+    .initialize("KLAVIYO_PUBLIC_API_KEY", applicationContext)
+    .registerForInAppForms()
+
+// ... or any time thereafter
+Klaviyo.registerForInAppForms()
+```
+
+### Behavior
+
+Once `registerForInAppForms()` is called, the SDK will load form data for your account and display no more than one form 
+within 10 seconds, based on form targeting and behavior settings.
```

Comment:
> no more than one form within 10 seconds

Is doing some heavy lifting to indicate we're not doing an ongoing listener without phrasing that in a negative way...

Reply:
```suggestion
within 15 seconds, based on form targeting and behavior settings.
```